playbooks/byo/openshift-cluster/metrics/uninstall_metrics.yml

```diff
@@ -0,0 +1,20 @@
+- hosts: localhost
```

Comment:
Most yml files in the repo begin with `---`, could you please add that for consistency?